src/main/java/com/amihaiemil/docker/Docker.java

```diff
@@ -32,8 +32,6 @@
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @since 0.0.1
- * @todo #1:30min Start implementing the other parts of the API,
```

Comment:
@llorllale I think you have to leave another puzzle for continuing implementation with, say, ``Images`` API.

Reply:
@amihaiemil yes